src/main/java/org/reactome/addlinks/fileprocessors/TargetPathogenFileProcessor.java

```diff
@@ -0,0 +1,83 @@
+/**
+ *
+ */
+package org.reactome.addlinks.fileprocessors;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Processes the Target Pathogen files. Currently, this file can be found here: https://target.sbg.qb.fcen.uba.ar/patho/reactome_map <br/>
+ * File structure is comma-separate values, with these fields: target_url, genome, target_genes, reactome_id, uniprot
+ * <br/>
+ * <br/>
+ * NOTE: "target_genes" is a pipe-separated list of gene symbols, BUT sometimes also contains commas (and sometimes spaces as well)
+ * with pipes such as "|RP-L40e,|RPL40," and "|APU18_05390| pNDM102337_153|groEL| MS6198_A148| CA268_28940| "
+ * and in some cases, it doesn't seem to exist at all.
+ * @author sshorser
+ *
+ */
+public class TargetPathogenFileProcessor extends FileProcessor<String>
+{
+
+	public TargetPathogenFileProcessor(String processorName)
+	{
+		super(processorName);
+	}
+
+	public TargetPathogenFileProcessor()
+	{
+		super(null);
```

Comment:
I'm a bit confused by the initiation of the class with `super(null)` here, with an additional super above that can be invoked